muxnetscan/main.c

```diff
@@ -624,15 +629,20 @@ int main(int argc, char *argv[]) {
     lv_timer_t *ui_refresh_timer = lv_timer_create(ui_refresh_task, UINT8_MAX / 4, NULL);
     lv_timer_ready(ui_refresh_timer);
 
+    pthread_create(&scan_networks_thread, NULL, scan_networks, NULL);
+    while (pthread_tryjoin_np(scan_networks_thread, NULL) != 0) {
+        lv_task_handler();
```

Comment:
this refresh is just while it is running the script to scan for available networks so will not cause any issues